docs/content.md

```diff
@@ -4,83 +4,79 @@
 
 ###  Global
```

Comment:
Would "Universal" work here? I see the connection for "Global" (e.g. global styles in CSS) but I think Universal as the category carries more of a "these are general guidelines to use throughout the system" without feeling a connection to development only.

Reply:
I think either would work. I wonder also if this whole section could be called Principles instead, like in Primer CSS, and we move Content to Foundations? 🤔 @ashygee